code/modules/research/server.dm

```diff
@@ -15,10 +15,6 @@
 	req_access = list(access_rd) //Only the R&D can change server settings.
 	circuit = /obj/item/weapon/circuitboard/rdserver
 
-/obj/machinery/r_n_d/server/New()
-	..()
-	Initialize()
-
```

Comment:
RDconsole (see Mark 1) will not find server in machinery list. Use `INITIALIZE_HINT_LATELOAD` in RDconsole

Reply:
Done.